torchvision/transforms/functional_tensor.py

```diff
@@ -48,3 +48,25 @@ def crop(img, top, left, height, width):
         raise TypeError('tensor is not a torch image.')
 
     return img[..., top:top + height, left:left + width]
+
+def to_grayscale(img, num_output_channels = 3):
```

Comment:
I think that we might need for now to be a bit more explicit, and maybe say `rgb_to_grayscale`, because we don't have the original colorspace information.